src/main/kotlin/com/asap/asapbackend/client/oauth/kakao/KakaoSocialLoginClient.kt

```diff
@@ -13,6 +13,10 @@ class KakaoSocialLoginClient(
     override fun handle(request: SocialLoginClient.Request): SocialLoginClient.Response {
         val kakaoUserInfo = kakaoOAuthClient.retrieveUserInfo(request.accessToken)
             ?: throw AuthException.AuthFailedException()
-        return SocialLoginClient.Response(kakaoUserInfo.id)
+        return SocialLoginClient.Response(
+            socialId = kakaoUserInfo.id,
+            email = kakaoUserInfo.getEmail(),
+            name = kakaoUserInfo.getName()
```

Comment:
```suggestion
            email = kakaoUserInfo.kakao_account.email,
            name = kakaoUserInfo.properties.nickname
```
이렇게 해도 될것같은데 get을 따로 구현하는 건 이유가 있는거야??

Reply:
약간 디미터의 법칙을 따를려고 했어

https://dkswnkk.tistory.com/687